### PR TITLE
FBX: Fix light and camera transformations and add some missing properties.

### LINKED ISSUE #851: Missing parameters for lights in FBX importing
I am importing fbx model files with lighting using assimp. Lights are arranged into the scene hierarchy and each light has transformation matrix. But direction vector is not present for the lights.

### PR DESCRIPTION
**Pull request by Sibras**
Fixes #851 #643 #807

### CODE REVIEW

File: code/FBXConverter.cpp [hunk 1]
```diff
@@ -708,10 +719,13 @@ void Converter::ConvertCamera( const Model& model, const Camera& cam )
...
 
     out_camera->mAspect = cam.AspectWidth() / cam.AspectHeight();
-    out_camera->mPosition = cam.Position();
-    out_camera->mUp = cam.UpVector();
-    out_camera->mLookAt = cam.InterestPosition() - out_camera->mPosition;
+    //cameras are defined along positive x direction
+    out_camera->mPosition = aiVector3D(0.0f);
+    out_camera->mLookAt = aiVector3D(1.0f, 0.0f, 0.0f);
+    out_camera->mUp = aiVector3D(0.0f, 1.0f, 0.0f);
     out_camera->mHorizontalFOV = AI_DEG_TO_RAD( cam.FieldOfView() );
```
Comment (Sibras) [inline code/FBXConverter.cpp]: All the necessary transform information is in the cameras node. So Fbx stores the cameras transformation in the scene graph as well as in the camera settings. This means you can either use one or the other but if you use both then the transform ends up getting applied twice resulting in incorrect values. Assimp already adds the required transformation info to the cameras node so the transform in the camera settings can be ignored.

I tested this with files exported from both blender and maya.

Comment (kimkulling) [inline code/FBXConverter.cpp]: OK thanks for the xplanation.

**Comment by kimkulling** [comment]
Merged!

**Comment by michaelAdsh** [comment]
hey 
what is the diffrent  between
PhysCamera001.Target  and  PhysCamera001 and  Camera001
how can i get my view camera matrix?
thx alot

**Pull request merged**